app/services/pokeRadar.js

```diff
@@ -0,0 +1,202 @@
+"use strict";
+
+const http = require('http'),
+      fs = require('fs'),
+      q = require('q'),
+      async = require('async'),
+      _ = require('underscore'),
+      coords = require(__base + 'resources/json/coords.json');
+
+//To point to currently used proxy in proxyList (0 is no proxy, 1 is first in list...)
+var proxyPointer = 0;
+//must be array of Strings in format "host:port"
+var proxyList = [];
+//holds coordinates of pokemon with at least 1 pokemon in them (required for global scan)
+if (scanType === 'global') {
+    var coordsFile = [];
+}
+//constructs URL for request to pokeradars API
+//pokeradar will return pokemon in the window of lat to lat+delta and lng to lng+delta
+const baseLink = function (lat, lng, delta) {
+    if (proxyPointer > 0) {
+        let hp = proxyList[proxyPointer - 1].split(':');
+        return {
+            host: hp[0],
+            port: hp[1],
+            path: 'http://www.pokeradar.io/api/v1/submissions?minLatitude=' + lat.toString() + '&maxLatitude=' + (lat + delta).toString() + '&minLongitude=' + lng.toString() + '&maxLongitude=' + (lng + delta).toString(),
+        };
+    } else {
+        return {
+            host: 'www.pokeradar.io',
+            path: '/api/v1/submissions?minLatitude=' + lat.toString() + '&maxLatitude=' + (lat + delta).toString() + '&minLongitude=' + lng.toString() + '&maxLongitude=' + (lng + delta).toString(),
+        };
+    }
+};
+
```

Comment:
This file could be made generic. References specific to pokeRadar could be moved out and could be made as function arguments. This way, the service may be used later for accessing other urls too.